src/components/TabBar/TabBar.tsx

```diff
@@ -16,45 +39,23 @@ type Props = {
 }
 type ElementProps = Omit<HTMLAttributes<HTMLDivElement>, keyof Props | 'role'>
```

Comment:
`ComponentProps<'div'>` にしといきたいやつ。

```suggestion
type ElementProps = Omit<ComponentProps<'div'>, keyof Props | 'role'>
```